.theia/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
I believe JSON doesn't support comments, can you confirm if this works? :o

Reply:
I don't know what that comment is doing there hahaha

Reply:
I added a space by mistake and canceled the change and apparently that appeared